config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
-	root 'application#hello'
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get 'rails_flavored_ruby/home'
+  get 'rails_flavored_ruby/help'
+	  root 'rails_flavored_ruby#home'
```

Comment:
- Lùi ra đầu dòng 4 spcaces thẳng hàng với get
- Dùng " thay cho ' tại toàn bộ file này